test/suites/ubuntu-core-edgex-image/build-gadget.sh

```diff
@@ -28,7 +29,7 @@ yq e -i '.defaults += {
       }
     }
   }
-} | .defaults.AZGf0KNnh8aqdkbGATNuRuxnt1GNRKkV.security-secret-store style="double"' gadget.yaml
+} | .defaults.AZGf0KNnh8aqdkbGATNuRuxnt1GNRKkV.security style="double"' gadget.yaml
```

Comment:
I think this is no longer necessary. I believe it was for setting `"off"` as a string.